src/main/java/com/bravo/user/service/AddressService.java

```diff
@@ -0,0 +1,59 @@
+package com.bravo.user.service;
+
+import com.bravo.user.dao.model.Address;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.AddressRepository;
+import com.bravo.user.dao.specification.AddressSpecification;
+import com.bravo.user.exception.DataNotFoundException;
+import com.bravo.user.model.dto.AddressReadDto;
+import com.bravo.user.model.filter.AddressFilter;
+import com.bravo.user.utility.PageUtil;
+import java.util.List;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AddressService {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AddressService.class);
+
+  private final AddressRepository addressRepository;
+  private final ResourceMapper resourceMapper;
+
+  public AddressService(AddressRepository addressRepository, ResourceMapper resourceMapper) {
```

Comment:
Random code convention note: when the other classes had constructors like this, their fields were final.  That is a @wheelers-websites convention ;)